build.gradle

```diff
@@ -74,31 +77,10 @@ subprojects {
     if (project.hasProperty('useMavenLocal') && project.useMavenLocal.toBoolean()) {
       mavenLocal()
     }
-    // QUESTION can we switch to jcenter() only?
-    mavenCentral()
-    maven {
-      name 'rubygems-release'
-      url 'http://rubygems-proxy.torquebox.org/releases'
-    }
-    maven {
-      name 'rubygems-prerelease'
-      url 'http://rubygems-proxy.torquebox.org/prereleases'
-    }
-  }
 
-  plugins.withType(JavaPlugin) {
-
-    project.tasks.withType(JavaCompile) { task ->
```

Comment:
I think this part has to stay.
I did a clean build and Gradle still seems to ignore this:
```
  sourceCompatibility = targetCompatibility = JavaVersion.VERSION_1_6
```
If I invoke this the output after building with Java 8 the class still seems to be built for Java 8 even with Gradle 2.13:
```
$ hexdump asciidoctorj-core/build/classes/main/org/asciidoctor/Asciidoctor.class|head
0000000 ca fe ba be 00 00 00 34 00 52 07 00 4e 07 00 4f
```